internal/storage/payload.go

```diff
@@ -176,6 +178,18 @@ func (w *PayloadWriter) AddBoolToPayload(msgs []bool) error {
 	return HandleCStatus(&status, "AddBoolToPayload failed")
 }
 
+func (w *PayloadWriter) AddByteToPayload(msgs []byte) error {
+	length := len(msgs)
+	if length <= 0 {
+		return errors.New("can't add empty msgs into payload")
+	}
+	cMsgs := (*C.int8_t)(unsafe.Pointer(&msgs[0]))
```

Comment:
better to be `uint8_t`.

Reply:
For almost every programming languages, `byte` corespond to `uint8_t`.

Reply:
Can we add a new data type  DataType_BLOB which means  []byte in go or  uint8 array in C?

And add AddUint8ToPayload methods.

@godchen0212 @DragonDriver

Reply:
Copied from mysql documentation.

In most respects, you can regard a BLOB column as a VARBINARY column that can be as large as you like. Similarly, you can regard a TEXT column as a VARCHAR column. BLOB and TEXT differ from VARBINARY and VARCHAR in the following ways:

1. For indexes on BLOB and TEXT columns, you must specify an index prefix length. For CHAR and VARCHAR, a prefix length is optional. See Section 8.3.5, “Column Indexes”.

2. BLOB and TEXT columns cannot have DEFAULT values.